docs/setup_guide.md

```diff
@@ -0,0 +1,120 @@
+
+### Deploying the module
+
+Now that we have a template the setup hardhat task can be used to deploy a DAO module instance. The setup task requires the following parameters `dao` (the address of the Safe), `oracle` (the address of the Realitio contract) and `template` (the template to used with Realitio). There are also optional parameters for more information run `yarn hardhat setup --help`.
+
+An example for this on Rinkeby would be:
+`yarn hardhat --network rinkeby setup --dao <safe_address> --oracle 0x3D00D77ee771405628a4bA4913175EcC095538da --template 0x0000000000000000000000000000000000000000000000000000000000000dad`
+
+This should return the address of the deployed DAO module. For this guide we assume this to be `0x4242424242424242424242424242424242424242`
+
+Once the module is deployed you should verify the source code. If you use a network that is Etherscan compatible and you configure the `ETHERSCAN_API_KEY` in your environment you can use the provided hardhat task to do this. 
+
+An example for this on Rinkeby would be:
+`yarn hardhat --network rinkeby verifyEtherscan --module 0x4242424242424242424242424242424242424242 --dao <safe_address> --oracle 0x3D00D77ee771405628a4bA4913175EcC095538da --template 0x0000000000000000000000000000000000000000000000000000000000000dad`
+
+### Enabling the module
+
+To allow the DAO module to actually execute transaction it is required to enable it on the Safe that it is connected to. For this it is possible to use the Transaction Builder on https://rinkeby.gnosis-safe.io. For this you can follow our tutorial on [adding a module](https://help.gnosis-safe.io/en/articles/4934427-add-a-module).
+
+## Snapshot integration
+
+Once the module is setup it is possible to configure a space on [Snapshot](https://snapshot.org/) to enable the DAO module plugin. For this the space configuration needs to include `"plugins": { "daoModule": { "address": "<module_address"} }`. An example for this can be found in the [🍯DAO space configuration](https://cloudflare-ipfs.com/ipfs/QmahDCSkdED9BLZ3VtH6aJ8P5TmvMYEfA7fJa4hGsvEpi2/).
+
+Once your space is configured you can attach transactions to you proposals via the plugin section:
+
+1. Open the plugin selection
+
+![Open the plugin selection](./snapshot_plugin_section.png)
+
+
+2. Add DAO module plugin
+
+![Add DAO module plugin](./snapshot_add_plugin.png)
+
+3. Add DAO module transaction
+
+<img src="./snapshot_module_add_tx.png"
+     alt="Add DAO module transaction"
+     width="250"/>
+<img src="./snapshot_module_tx_details.png"
+     alt="Enter transactiond etails"
+     width="250" />
+<img src="./snapshot_module_tx_confirm.png"
+     alt="Check transaction details"
+     width="250"/>
+
+4. Check preview of transactions
+
+![Transactions preview](./snapshot_plugin_preview.png)
+
+Once the proposal has been resolved it is possible to use the plugin to submit the proposal to the DAO module via the plugin. 
```

Comment:
The plugin is the same, right?
```suggestion
Once the proposal has been resolved it is possible to submit the proposal to the DAO module via the newly added plugin. 
```

Reply:
I would leave out `newly added` as this step normally happens after the proposal has been resolved (so 7 days later :D)